From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Huangbin Zhan <zhanhb88@gmail.com>
Date: Fri, 18 Feb 2022 14:19:23 +0800
Subject: [PATCH] tools: disable TOOLS_FIT_FULL_CHECK

	U-Boot disallows unit addresses by default. Disable TOOLS_FIT_FULL_CHECK
	to allow at symbol in node names.

https://github.com/openwrt/openwrt/commits/master/scripts/mkits.sh
https://github.com/u-boot/u-boot/commit/3f04db891a353f4b127ed57279279f851c6b4917
---
 tools/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/Kconfig
+++ b/tools/Kconfig
@@ -31,7 +31,7 @@ config TOOLS_FIT
 	  Enable FIT support in the tools builds.
 
 config TOOLS_FIT_FULL_CHECK
-	def_bool y
+	bool "Do a full check of the FIT"
 	help
 	  Do a full check of the FIT before using it in the tools builds
 
